BOJ/11057.오르막 수/Munang.py

```diff
@@ -0,0 +1,16 @@
+
+
+n = int(input())
+
+dp = [[0 for i in range(10)]for _ in range(0,n+1)]
+
+for i in range(0,n+1):
+    for j in range(10):
+        if(i==1):
+            dp[i][j] =1
+        else:
+            for k in range(10):
+                if(k<=j):
+                    dp[i][j] = dp[i-1][j-k]+dp[i][j]
```

Comment:
```suggestion
        if(i==1):
            dp[i][j] =1
            continue;
        for k in range(10):
            if(k>j): continue;
            dp[i][j] = dp[i-1][j-k]+dp[i][j]
```
이런식으로 If로 진입하여 무언가를 실행하기보단, If 진입시 continue/break/return 등을 사용하는 것을 추천드려요!

관련자료: https://velog.io/@kskim/%ED%95%A8%EC%88%98%EC%9D%98-%EB%8B%A8%EC%9D%BC-%EC%B1%85%EC%9E%84-%EC%A1%B0%EA%B1%B4%EC%9D%84-%EC%B6%A9%EC%A1%B1%EC%8B%9C%EC%BC%9C%EB%9D%BC-else%EB%AC%B8%EC%9D%84-%EB%B0%B0%EC%A0%9C%ED%95%B4%EB%9D%BC

Reply:
글 잘 참고하겠습니다 ! 알고있는 내용인데 적용하기가 어려웠는데요. 글로 보니까 더 좋은것 같습니다 ! 감사합니다.